sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/LoadBalancer.java

```diff
@@ -130,7 +131,7 @@
              * @param name the name for the frontend
              * @return the first stage of a new frontend definition
              */
-            LoadBalancerPublicFrontend.DefinitionStages.Blank<WithCreate> definePublicFrontend(String name);
+            LoadBalancerPublicFrontend.DefinitionStages.Blank<WithCreateAndOutboundRule> definePublicFrontend(String name);
```

Comment:
only allow define outbound rule after defining public frontend because outbound rule can only be associate with public frontend.